platform/android/src/native_map_view.cpp

```diff
@@ -682,13 +682,15 @@ jni::Local<jni::Object<TransitionOptions>> NativeMapView::getTransitionOptions(J
     const auto transitionOptions = map->getStyle().getTransitionOptions();
     const auto duration = std::chrono::duration_cast<std::chrono::milliseconds>(transitionOptions.duration.value_or(mbgl::Duration::zero())).count();
     const auto delay = std::chrono::duration_cast<std::chrono::milliseconds>(transitionOptions.delay.value_or(mbgl::Duration::zero())).count();
-    return TransitionOptions::fromTransitionOptions(env, duration, delay);
+    const auto enablePlacementTransitions = static_cast<const jboolean>(transitionOptions.enablePlacementTransitions);
```

Comment:
`static_cast<const jboolean>(transitionOptions.enablePlacementTransitions);`can be simplified to 
 `(jboolean) transitionOptions.enablePlacementTransitions`